packages/cli/README.md

```diff
@@ -3384,6 +3384,20 @@ COMMON FLAGS
 
 DESCRIPTION
   unenroll a hub from a channel
+
+EXAMPLES
+  prompt user for hub and channel and then unenroll hub from channel
+
+    $ smartthings edge:channels:unenroll
+
+  prompt user for a channel the given hub is enrolled in and then unenroll it
+
+    $ smartthings edge:channels:unenroll 2fa0912f-cb73-424e-97f9-ffff76ea4f2a
+
+  unenroll the hub from the specified channel
```

Comment:
What is the difference between these two examples? The first seems to be showing unenrolling a hub from a channel, and the second is "unenroll[ing] the hub from the specified channel".

Reply:
The first example doesn't provide any arguments, so the user will be shown a list of hubs and prompted to choose one, then shown a list of channels that hub is subscribed to and prompted to choose one.  The second example gives one argument -- the UUID of a hub -- so it'll skip straight to asking for the channel to unsubscribe that specific hub from. The third example specifies the UUID of both the hub and the channel to unenroll it from, so the user won't be prompted for anything else.